<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="selectTabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Home</string>
     </attribute>
     <widget class="QTableWidget" name="historyTableWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>791</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="generateButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>88</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Gerar Hash</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>771</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="baseTextLayout">
       <item>
        <widget class="QLabel" name="inputLabel">
         <property name="text">
          <string>Texto base:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="inputLineEdit"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>60</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="algorithmSelectorLayout">
       <item>
        <widget class="QLabel" name="algorithmSelectorLabel">
         <property name="text">
          <string>Algoritmo:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="algorithmComboBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>771</width>
        <height>41</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="outputLayout">
       <item>
        <widget class="QLabel" name="outputLabel">
         <property name="text">
          <string>Hash resultante:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="hashOutputLineEdit">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="clearHistoryButton">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Limpar hist√≥rico</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Detalhes do Algoritmo</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
